### PR TITLE
Blog App: validations and model specs

### PR DESCRIPTION
**Pull request by VaheStepanyan100**
- Added the following validations:
  - For the User model:
    - Name must not be blank.
    - PostsCounter must be an integer greater than or equal to zero.
  - For the Post model:
    - Title must not be blank.
    - Title must not exceed 250 characters.
    - CommentsCounter must be an integer greater than or equal to zero.
    - LikesCounter must be an integer greater than or equal to zero.
- Added unit specs for all of your models' methods and validations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ If you like this project, give it a ⭐️!
```
Comment (HAMMAS-SALEEM) [inline README.md:134]: **[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @VaheStepanyan100 👋

Great work on implementing the validations & specs. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following Requirements
✔️ Great work on making the app work as expected 👍
✔️ Great work on covering all the edge cases in tests 🔥

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
